cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-what-is-rosa.adoc

```diff
@@ -61,12 +61,8 @@ Autoscaling allows you to automatically adjust the size of the cluster based on
 
 === Maximum number of worker nodes
 The maximum number of worker nodes in ROSA clusters versions 4.14.14 and later is 249. For earlier versions, the limit is 180 nodes.
-ifdef::temp-ifdef[]
-See xref:../../rosa_planning/rosa-limits-scalability.adoc#rosa-limits-scalability[limits and scalability] for more details on node counts.
-endif::[]
-ifdef::temp-ifdef[]
-See xref:../../rosa_planning/rosa-hcp-limits-scalability.adoc#rosa-hcp-limits-scalability[limits and scalability] for more details on node counts.
-endif::[]
+// Removed as part of OSDOCS-13310, until figures are verified.
```

Comment:
How can I verify this. Do you have a preview link?

Reply:
Removed from https://88263--ocpdocs-pr.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-what-is-rosa

Reply:
Thanks. :heavy_check_mark: